.github/workflows/build-and-deploy-storybook.yml

```
@@ -52,3 +52,17 @@ jobs:
       - run: pnpm deps
       - run: pnpm build
       - run: pnpm test
+
+  Build_Microfrontend:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - uses: pnpm/action-setup@v4
+      - run: pnpm deps
+      - run: pnpm build
+      - run: |
+          cd apps/gitness
+          pnpm install
+          pnpm run build:webpack
```

Instead of cd'ing to the directory, you can just filter from the top
```suggestion
      - run: pnpm -r -F ./apps/gitness build:webpack
```

---

Or, since this is the only one that does use `build:webpack`, you can simplify more
```suggestion
      - run: pnpm -r build:webpack
```